### DISCUSSION TITLE
Specify how the buffer could be increased in error messages

### DISCUSSION THREAD

**Discussion by marklit** [duckdb/duckdb#8102] [General]
I'm getting the following back from a SELECT statement reading from a large CSV file and doing some light transformations on it. Without the transformations, the job runs without issue but with the transformations come up.

```
Error: near line 5: Invalid Input Error: Line does not fit in one buffer. Increase the buffer size.
```

Could the software give an example of how this buffer would be increased? Googling around doesn't give me an obvious answer nor does the current CSV docs.

My ideal would be something like:

```
Error: near line 5: Invalid Input Error: Line does not fit in one buffer. Increase the buffer size. e.g. SET memory_limit='10GB';
```

**Comment by aaronlutz** [comment]
This would be nice because on this page https://duckdb.org/docs/sql/configuration.html when you search for buffer you find the `arrow_large_buffer_size` setting, which does not resolve this issue.

**Comment by Tishj** [comment]
The setting you're looking for is specific to CSV loading, it's described here:
https://duckdb.org/docs/data/csv/overview.html

The setting is called `buffer_size`

I agree however that this is assuming too much knowledge of the system.
Maybe the error should include a minimal example
